normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -47,6 +47,10 @@ export class RecipeForm extends React.Component {
 
     return this.validateForm(combinedFormValues)
     .catch(error => {
+      dispatch(showNotification({
```

Comment:
How hard would testing this be? Could we use a simulated click with enzyme to trigger the submit?